src/main/java/com/kosherjava/zmanim/hebrewcalendar/HebrewDateFormatter.java

```diff
@@ -116,13 +116,13 @@ public class HebrewDateFormatter {
 	 *  &#x05E9;&#x05D5;&#x05D1;&#x05D4;,&#x05E9;&#x05D9;&#x05E8;&#x05D4;,&#x05D4;&#x05D2;&#x05D3;&#x05D5;&#x05DC;,
 	 *  &#x05D7;&#x05D6;&#x05D5;&#x05DF;,&#x05E0;&#x05D7;&#x05DE;&#x05D5;"</code>
 	 */
-	private EnumMap<JewishCalendar.Parsha, String> hebrewParshaMap;
+	private final EnumMap<JewishCalendar.Parsha, String> hebrewParshaMap;
 	
 	/**
 	 * Default constructor sets the {@link EnumMap}s of Hebrew and default transliterated parshiyos.
 	 */
 	public HebrewDateFormatter() {
-		transliteratedParshaMap = new EnumMap<JewishCalendar.Parsha, String>(JewishCalendar.Parsha.class);
+		transliteratedParshaMap = new EnumMap<>(JewishCalendar.Parsha.class);
```

Comment:
As of Java 7, the diamond operator <> does not need explicit types. See here: https://stackoverflow.com/a/5999744